packages/useful-types/README.md

```diff
@@ -0,0 +1,16 @@
+# `@shopify/useful-types`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Fuseful-types.svg)](https://badge.fury.io/js/%40shopify%2Fuseful-types.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/useful-types.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/useful-types.svg)
+
+A few handy TypeScript types.
+
+## Installation
+
+```bash
+$ yarn add @shopify/useful-types
+```
+
+## Usage
+
+Review the type definitions from this library to see what they can do for you.
```

Comment:
I would definitely prefer to see actual explanations for each type. Parsing a complex type definition is hard enough for experienced typescript users, let alone people less familiar.